config/secrets.yml

```diff
@@ -30,11 +30,13 @@ common: &common
   redis_namespace: <%= ENV["REDIS_NAMESPACE"] %>
 
 development:
-  secret_key_base: <%= ENV["KEY_DEV"] %>
+  secret_key_base: 1b7333657709a8e14d929c518b4befc85655aa71c0f9bd09250123f1d5b2273c0b21c4399e9a68c8b72c356b48475ac856efce63deb08781505a691f76a95aec
```

Comment:
this shouldnt be in a .env file? 🤔

Reply:
For development it's not really needed (this key is used for your tests only).
For production it must be as an env variable.